src/cortexpy/tesserae.py

```diff
@@ -75,14 +76,15 @@ def __init__(self, pdel=DEFAULT_DEL, peps=DEFAULT_EPS, prho=DEFAULT_REC, pterm=D
     def __initialize(self, query, targets):
         self.nseq = len(targets) + 1
         self.maxl = max_length(query, targets)
+        self.qlen = len(query)
 
-        self.vt_m = np.full([self.nseq, self.maxl + 1, self.maxl + 1], SMALL, dtype=np.float64)
-        self.vt_i = np.full([self.nseq, self.maxl + 1, self.maxl + 1], SMALL, dtype=np.float64)
-        self.vt_d = np.full([self.nseq, self.maxl + 1, self.maxl + 1], SMALL, dtype=np.float64)
+        self.vt_m = np.full([self.nseq, self.maxl + 1, self.qlen + 2], SMALL, dtype=np.float64)
```

Comment:
Simple yet effective change.  Perhaps the +1 to maxl and +2 to qlen should be moved into lines 78 and 79 to make them symmetric with the nseq + 1 in line 77.  Though I need to read on to see if that would screw anything up too badly.

Also, why is it qlen + 2 and not qlen + 1?